### ISSUE TITLE
Accessibility Feedback for Design Intent Review: LGY: Certificate of Eligibility

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23562]
# Accessibility Feedback for Design Intent Review: LGY: Certificate of Eligibility

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of contact
Josh, Trevor, and Noah

## Must
- [developers] ensure that the list loop pattern for previous military history accommodates unique accessible labels for past service fields. For example, if there are multiple "branch of service" dropdowns each of them must have a unique accessible label so screen reader users can easily tell them apart. 

## Should
- For the question "have you received a purple heart", I strongly recommend providing a brief note or reason explaining _why_ the VA is asking for this or why they don't already know this. During our PTSD research, VSOs and doctors both noted that questions like this that do not explain their purpose may be taken as questioning a Veteran's service or cause anxiety e.g. "if I don't have a purple heart, does that mean I'm not going to be eligible?"
- Reword `please go to your profile page` link to include the purpose of said link. For example, `Go to your profile page to update your contact information` provides both a purpose and a destination.
- Step 3 of 4: Your VA loan history should have a [Veteran's loans written as a description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).
- Split up Step 1 of 4: Your VA loan history into multiple steps since it currently has too many questions including: date fields, address fields, and yes/no fields 
- [developers] add `aria-hidden="true"` to the "!" icon if it is kept in the final version. The bolded "note:" provides a text equivalent, so the "!" should be hidden to screen reader users.

## Consider
- Using more descriptive `h2`s to indicate states post log in. This will make it more obvious to screen reader Veterans that information on the page has been changed as ~70% of screen reader users will use headings to scan or navigate through the page. It will also make it more clear to Veterans with low vision, cognitive disabilities, and many more.
- Adding the file size of the Certificate of Eligibility as well for Veterans with low bandwidth, limited data plans, etc. VA.gov estimates around 25% of Veterans live in rural areas, but recent figures are often cited as high as 45% (David Shulkin). Providing the file size could help more marginalized Veteran groups when and how they want to download.  
- Removing "Click on the link below" in favor of just "View, print, or save your Certificate of Eligibility." This small language changes can make the experience more inclusive to Veterans who don't use a mouse to navigate VA.gov (e.g. keyboard, mobile users)
- Redesigning the "missing information" state to have a stronger gestalt and provide more information on what the Veteran is missing in particular. I recommend exploring this on mobile as opposed to desktop as having less space above the fold and a narrower view port may impact what components are selected e.g. the grey box note's padding will squeeze the text, so it may be undesired. See examples below.

![image](https://user-images.githubusercontent.com/14154792/115565249-377aa000-a287-11eb-9a2a-ae83397773b2.png)

**Comment by 1Copenut** [comment]
VSP agrees with @joshkimux feedback!

**Comment by tblackwe** [comment]
Discussed this ticket with @data-doge 
Adding to team 🏖️ to review if still applicable

**Comment by tblackwe** [comment]
closing in favor of #53279 as that ticket has more research done by engineering team.
